src/ParseServer.js

```diff
@@ -240,7 +240,8 @@ class ParseServer {
       });
       // verify the server url after a 'mount' event is received
       /* istanbul ignore next */
-      api.on('mount', function () {
+      api.on('mount', async function () {
+        await new Promise(resolve => setTimeout(resolve, 1000));
```

Comment:
- Why did you choose 1s?
- Is the reason for this change in the PR title?

Reply:
There is no other express event other than mount, and the server is not reachable until `httpServer.listen` is called. How do you think we should approach this?